app/views/job_templates/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= include_javascript %>
+<%= include_javascript if SETTINGS[:version].short == '1.9' %>
```

Comment:
The scope of the plugin makes me think these kinds of checks are going to get out of control really fast. Can we just not move to only supporting nightly now? 

X should get bumped whenever incompatibility is introduced - which tends to be every release if you don't incorporate these checks.

For us, that means  `1.0` would support `Foreman 1.10`, then `2.0` would support `Foreman 1.11`, and so on.  That's how foreman_salt and foreman_bootdisk generally work, and I think it works well.

If we want to support more than one version at a time, the most important bug fixes can go to the previous version's branch. Nearly all other Foreman plugins do it this way, unless its trivial to have dual-version compatability, but I don't think that's the case for us.